### ISSUE TITLE
0.3 not building due to non-public file

### ISSUE THREAD

**Issue by MattSich** [prebid/prebid-mobile-ios#73]
hi @ppuviarasu !

on 0.3 `PBServerRequestBuilder.h` was made public but it can't be found when pulling in the project as a pod. After a bit more digging, turns out that the imported pod project has `PBServerRequestBuilder.h` set to project scope. 
I think since `PBServerRequestBuilder.h` is referencing `PBAdUnit.h` which was not changed from public to project, `PBServerRequestBuilder.h` can't be public and therefore is set to project when used with cocoapods. 
Can we roll back the `PBAdUnit.h` access change and make it public again? 

thanks

**Comment by ppuviarasu** [comment]
Hey Matt,

Thanks for bring this to our notice... we will work on this issue & provide updates soon.

Thanks

**Comment by shujew** [comment]
@MattSich I fixed it temporarily by adding this to the end of my Podfile

```
post_install do |installer|
    installer.pods_project.targets.each do |target|
        # Fix for Prebid v0.3 missing header file issue
        for i in 0..target.headers_build_phase.files.length - 1
            build_file = target.headers_build_phase.files[i]
            if build_file.display_name == 'PBServerRequestBuilder.h'
                build_file.settings = { 'ATTRIBUTES' => ['Public']}
            end
        end
    end
end
```

**Comment by qkzhu** [comment]
Same issue here.  Why not simply make `PBServerRequestBuilder.h ` public or remove it from umbrella header file?

@shujew this will work, but it is no difference compare with manually change it to public. And if I want make `PrebidMobile` as my library dependency, each user use my library will have to use this fix, not good. Hope your can fix this soon.

**Comment by ppuviarasu** [comment]
This is fixed in 0.5 version.

Please raise a issue if it still exists
